### PR TITLE
[Actions] Orbit

### LINKED ISSUE #3711: Orbit
### App
Orbit

### Triggers
* New Activity
* New Member

### Actions
* Create Activity
* Create a Member
* Create a Member Note
* Get a Member by Identity

### PR DESCRIPTION
**Pull request by vellames**
Closes #3711

Unfortunately, Orbit is presenting an error in their API for webhook creation. Even trying to create a Webhook directly from their site, I receive a `422` error.

### CODE REVIEW

File: components/orbit/actions/create-activity/create-activity.mjs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      description: "A custom weight to be used in filters and reports; defaults to 1.",
+      optional: true,
+    },
+    activityTypeKey: {
+      type: "string",
+      label: "Activity Type Key",
+      description: "The key for a custom activity type for the workspace. Will create a new activity type if it does not exist.",
+      optional: true,
+    },
+    ocurredAt: {
```
Comment (alysonturing) [inline components/orbit/actions/create-activity/create-activity.mjs:61]: You have a typo here, `occurred`

File: components/orbit/actions/create-activity/create-activity.mjs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      activity_type_key: this.activityTypeKey,
+      occurred_at: this.ocurredAt,
+      properties: this.properties,
+    };
+    const res = await this.app.createActivity(
+      this.workspaceSlug,
+      this.workspaceMemberSlug,
+      activity,
+    );
+    $.export("$summary", "Activity successfully created");
```
Comment (alysonturing) [inline components/orbit/actions/create-activity/create-activity.mjs:90]: can you export the identification of the created resource?

File: components/orbit/actions/create-member-note/create-member-note.mjs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  async run({ $ }) {
+    const note = {
+      body: this.note,
+    };
+    const res = await this.app.createNote(
+      this.workspaceSlug,
+      this.workspaceMemberSlug,
+      note,
+    );
+    $.export("$summary", "Note successfully created");
```
Comment (alysonturing) [inline components/orbit/actions/create-member-note/create-member-note.mjs:41]: can you export the identification of the created resource?

File: components/orbit/actions/create-member/create-member.mjs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+      twitter: this.twitter,
+      email: this.email,
+      linkedin: this.linkedin,
+      devto: this.devto,
+    };
+    const res = await this.app.createMember(
+      this.workspaceSlug,
+      member,
+    );
+    $.export("$summary", "Member successfully created");
```
Comment (alysonturing) [inline components/orbit/actions/create-member/create-member.mjs:150]: can you export the identification of the created resource?

File: components/orbit/actions/get-member-by-identity/get-member-by-identity.mjs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      source_host: this.sourceHost,
+      uid: this.uid,
+      username: this.username,
+      email: this.email,
+    };
+    const res = await this.app.getMemberByIdentity(
+      this.workspaceSlug,
+      query,
+    );
+    $.export("$summary", "Member(s) successfully fetched");
```
Comment (alysonturing) [inline components/orbit/actions/get-member-by-identity/get-member-by-identity.mjs:69]: can you export the count number of exported data?

**Review by alysonturing — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please check the minor changes pointed out.

**Review by alysonturing — APPROVED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Orbit_3711_1300.pdf](https://github.com/PipedreamHQ/pipedream/files/10878650/Orbit_3711_1300.pdf)

**Comment by vellames** [comment]
Hey @vunguyenhung. I have tested the find and looks like this is a problem in Orbit API. Try to add a github account to one of the members and use the filters: `Source = github` and `Username = {yourusername}`.
I ensured I am using all params in the right way and looks like this is something related to the way that Orbit works

**Comment by vunguyenhung** [comment]
Hi @vellames, your points make sense to me. I'll move this to ready for release!

**Pull request merged**